### ISSUE TITLE
prefer-default-export fails to recognize export imports

### ISSUE THREAD

**Issue by krainboltgreene** [benmosher/eslint-plugin-import#749]
``` js
// file: utilities/log/index.js.js

export default console.log
```

``` js
// file: utilities/index.js

export {default as log} from "./log"
```

Example: 
<img width="505" alt="screen shot 2017-02-18 at 2 41 10 am" src="https://cloud.githubusercontent.com/assets/334809/23092615/b8f4e484-f583-11e6-8e61-a8e60022cbbe.png">

**Comment by ljharb** [comment]
That rule is to prefer a default export. `export from` proxies a default import to a named export. It is correct for the rule to complain there.

**Comment by krainboltgreene** [comment]
Good to know, what's the solution here?

**Comment by ljharb** [comment]
@krainboltgreene pending the [proposal](https://github.com/leebyron/ecmascript-export-default-from) to allow it, you'll need to do it in two lines. `import logger from './logger'; export default logger`

**Comment by ljharb** [comment]
Although, reading that document, it looks like `export { default } from "./logger";` would actually work for you!

**Comment by krainboltgreene** [comment]
Hmm, I guess that works. In this case this file will in the future have multiple exports.
